wsf_scraping/pipelines.py

```diff
@@ -54,18 +54,18 @@ def check_keywords(self, item, spider_name, base_pdf_path):
         download_only = self.settings['DOWNLOAD_ONLY']
         feed = self.settings['FEED_CONFIG']
         pdf_result_path = os.path.join(
-            os.path.curdir,
             'results',
             'pdfs',
             spider_name,
+            os.path.basename(item['pdf']),
         )
 
         # Convert PDF content to text format
-        with open(base_pdf_path, 'rb') as f:
+        with open(item['pdf'], 'rb') as f:
             if download_only:
                 os.rename(
-                    ''.join(['/tmp/', item['pdf']]),
-                    ''.join([pdf_result_path, item['pdf']])
+                    item['pdf'],
```

Comment:
so are you now saving the temp file in the current directory instead?

Reply:
if so is there any way to delete tmp files if scrapy fails? we can add this as an issue